tests/test_unit_pto_detector_get_is_pto.py

```diff
@@ -0,0 +1,17 @@
+import os, sys
+from unittest import mock
+from unittest.mock import patch
+from pto_detector import pto_detector
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+@patch('pto_detector.pto_detector.get_is_pto')
```

Comment:
@rahul-bhave - this should patch requests.post not the get_is_pto method :) .... get_is_pto is making exactly one external call. You need to mock that.

Effectively what you would be doing by mocking out requests.post is to say "If requests.POST works fine, then I can assure you that the method get_is_pto is working right". You should be checking this method by setting the returned score to be both 1 and not 1.

Reply:
I have resolved this comment by mocking out requests library.

Reply:
@qxf2 , I have resolved your comments also, removed unittest dependency from the tests as discussed in sprint meeting. Can I share this for PR for review now?